docs/docsite/rst/community/steering/community_steering_committee.rst

```diff
@@ -117,7 +117,11 @@ When reviewing community collection `inclusion requests <https://github.com/ansi
 
 #. In the course of the review, the Committee member marks items as completed or leaves a comment saying whether the reviewer expects an issue to be addressed or whether it is optional (for example, it could be **MUST FIX:** <what> or **SHOULD FIX:** <what> under an item).
 
-#. For a collection to be included in the Ansible community package, the collection must be reviewed and approved by at least two Committee members.
+#. For a collection to be included in the Ansible community package, the collection:
+
+  * Must be reviewed and approved by at least one Steering Committee member and one non-Steering Committee person.
+  * Non-Steering Committee reviews to be counted for inclusion must be checked and approved by another Steering Committee member.
+  * Reviewers must not be involved in development of the collection.
```

Comment:
Reason: in the past, reviewers sometimes created PRs to fix smaller issues. Once these get merged they have been involved in the development of the discussion and thus their review would be disqualified :)
```suggestion
  * Reviewers must not be involved significantly in development of the collection.
```
I would add though that they must declare potential conflicts of interest:
```suggestion
  * Reviewers must not be involved significantly in development of the collection. They must declare any potential conflict of interest (for example, being friends/relatives/coworkers of the maintainers/authors, being users of the collection, or having contributed to that collection recently or in the past).
```